contents/hackage.mdx

```diff
@@ -57,12 +58,39 @@ remote-repo: mirror:{{http_protocol}}{{mirror}}
 
 注意，此处的注释是两条短线`--`.
 
-再执行`cabal update`, 即可使用`cabal`安装包了。
+再执行`cabal v2-update`, 即可使用`cabal`安装包了。
```

Comment:
`cabal v2-update` 和 `cabal update` 的区别是？

Reply:
这个对于目前的 cabal 确实没有必要了

不过具体的区别，在以前 (cabal <= v3.0.0.0)，v2 是新的方式，和老的更新/安装机制区分，我只是习惯了，等下改掉

```default
 [new-style projects (forwards-compatible aliases)]
  Since cabal-install-3.0.0.0, all 'v2-' prefixed names of commands are just
  aliases for the simple unprefixed names.  So v2-build is an alias for
  build, v2-install for install and so on.
```